Utilities/bootstrap

```diff
@@ -0,0 +1,99 @@
+#!/bin/sh
+
+
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2019 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+#
+# -------------------------------------------------------------------------
+# This script is used to bootstrap the tools-support-core package from its
```

Comment:
We should call this `import` not `bootstrap`.

Bootstrap in our world is mostly used to refer to the process by which a tool which can build itself (e.g. a compiler or a build system) is initially built. As in, "pulling yourself up by your boot straps".

Reply:
Makes sense. I wanted to follow the scheme from the other swift repositories. But you're right, it's not building the project.